Cargo.toml

```diff
@@ -24,5 +24,5 @@ default-features = false
 [features]
 default = []
 std = ["parity-hash/std", "bigint/std"]
-strict = []
+strict = ["wasm-libc/strict", "wasm-alloc/strict"]
```

Comment:
:+1:

I'm wondering, it is possible to remove these lines

```yml
   - cargo build --verbose --features strict --manifest-path=alloc/Cargo.toml
   - cargo build --verbose --features strict --manifest-path=libc/Cargo.toml
```

from .travis.yml with this change?

Reply:
aye